java/libphonenumber/pom.xml

```diff
@@ -31,6 +31,7 @@
     <plugins>
       <plugin>
         <artifactId>maven-jar-plugin</artifactId>
+        <version>2.2</version>
```

Comment:
These are just the version numbers my setup was using by default I suppose, I don't have a strong preference if you want different versions. This works.